tools/make/golang.mk

```diff
@@ -42,17 +44,10 @@ go.clean: ## Clean the building output files
 	@echo "===========> Cleaning all build output"
 	@rm -rf $(OUTPUT_DIR)
 
-.PHONY: go.format.verify
-go.format.verify:
-ifeq ($(shell which goimports), )
-	@echo "===========> Installing missing goimports"
-	@go get -v golang.org/x/tools/cmd/goimports
-	@go install golang.org/x/tools/cmd/goimports
-endif
-
-.PHONY: go.format
-go.format:  go.format.verify
-	@echo "===========> Running go codes format"
-	@gofmt -s -w .
-	@goimports -w -local $(ROOT_PACKAGE) .
-	@go mod tidy
+.PHONY: go.tidy
```

Comment:
you also might want to edit the top level `format` target and reuse `go.tidy` there
https://github.com/envoyproxy/gateway/blob/80b78e5521a0ce32274c36a845da3955df4aec09/Makefile.targets.mk#L76

Reply:
Done. Please review again.